src/Kernels/Local.h

```diff
@@ -80,17 +80,17 @@ class seissol::kernels::Local : public LocalBase {
                                         ConditionalIndicesTable& indicesTable,
                                         kernels::LocalData::Loader& loader,
                                         double time,
-                                        double timeStepWidt);
+                                        double timeStepWidth);
 
     void flopsIntegral(FaceType const i_faceTypes[4],
                        unsigned int &o_nonZeroFlops,
                        unsigned int &o_hardwareFlops );
                         
     unsigned bytesIntegral();
 
-    void setDamageParameters( seissol::initializer::parameters::DamagedElasticParameters* iDamagedElasticParameters ) {
-      mDamagedElasticParameters = iDamagedElasticParameters;
-    }
+    void const setDamagedElasticParameters(seissol::initializer::parameters::DamagedElasticParameters* mDamagedElasticParameters){
```

Comment:
Why `void const`?

Reply:
It must have been left there as a mistake during the merge. Modified